configs/coins/zcash.json

```diff
@@ -22,10 +22,10 @@
     "package_name": "backend-zcash",
     "package_revision": "satoshilabs-1",
     "system_user": "zcash",
-    "version": "4.2.0",
-    "binary_url": "https://z.cash/downloads/zcash-4.2.0-linux64-debian-stretch.tar.gz",
+    "version": "4.3.0",
+    "binary_url": "https://download.z.cash/downloads//zcash-4.3.0-linux64-debian-stretch.tar.gz",
```

Comment:
```suggestion
    "binary_url": "https://download.z.cash/downloads/zcash-4.3.0-linux64-debian-stretch.tar.gz",
```
Probably a typo.

Reply:
BTW why the URL change?

Reply:
Didn’t notice the typo. Thnx will add fix. As for the url I’m not really sure, If you go to the official zcash website this is the one you’re redirected for downloads. https://download.z.cash/downloads/ even original one redirects you here. Assumed they changed it but looks like both are working.